build_library/toolchain_util.sh

```diff
@@ -27,6 +28,9 @@ BOARD_PROFILES["amd64-usr"]="coreos:coreos/amd64/generic"
 BOARD_CHOSTS["arm64-usr"]="aarch64-cros-linux-gnu"
 BOARD_PROFILES["arm64-usr"]="coreos:coreos/arm64/generic"
 
+BOARD_CHOSTS["ppc64le-usr"]="powerpc64le-unknown-linux-gnu"
```

Comment:
YES. This should be changed as you suggested.

On April 29 that I wrote this patch, crossdev -t powerpc64le-cros-linux-gnu can't build cross toolchain for ppc64le. But this time, gentoo community enabled ppc64le platform for crossdev.